docs/user/gcp/quota.md

```diff
@@ -0,0 +1,13 @@
+# Handling quota on GCP
+
+All cloud providers have some concept of limits imposed to protect their services from misuse as well as protect customers from mistakes that can leads to excessive charges.  In GCP these will vary by region, project and account.  This is great news in terms of flexibility but can present an initial hurdle if the defaults in your project are too low.
+
+## Common problems
```

Comment:
can you split this into 2 sections: Usage for a cluster, and Common problems.

I think the Usage should provide details on what's required and then users are empowered to check their limits. common problems are useful to draw attentions to certain resources.

Reply:
works for me.